conn.go

```diff
@@ -695,35 +714,31 @@ func (c *Conn) Handshake() Alert {
 	var alert Alert
 	if c.hState == nil {
 		logf(logTypeHandshake, "%s First time through handshake (or after stateless retry), setting up", label)
-		alert = c.HandshakeSetup()
-		if alert != AlertNoAlert {
+		alert := c.HandshakeSetup()
```

Comment:
This colon is unnecessary, since `alert` is declared above.

Reply:
I moved the variable declaration instead, prefer to keep variables as local as possible.